Mathlib/LinearAlgebra/Transvection.lean

```diff
@@ -24,54 +27,76 @@ public import Mathlib.RingTheory.TensorProduct.IsBaseChangeHom
 
 namespace LinearMap
 
+open Module
+
 variable {R V : Type*} [CommSemiring R] [AddCommMonoid V] [Module R V]
 
 /-- The transvection associated with a linear form `f` and a vector `v`.
 
 NB. It is only a transvection when `f v = 0`. See also `Module.preReflection`. -/
-def transvection (f : Module.Dual R V) (v : V) : V →ₗ[R] V where
+def transvection (f : Dual R V) (v : V) : V →ₗ[R] V where
```

Comment:
The docstring here is really confusing. Can you clarify the NB? Looking at the definition `LinearEquiv.transvection`, and if I understand the NB correctly, I think `transvection` is the wrong name here, unless that is established language. But if you were to change the name here, that's a separate PR.

Reply:
I tried to be more explicit in the header of the file.